src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,85 @@
+package calculator;
+
+import calculator.type.TokenType;
+import calculator.vo.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Calculator {
+    public Answer add(String text) {
+        UserInput userInput = getUserInput(text);
+
+        var expression = userInput.toExpression();
+
+        if (validateExpression(expression) == false)
```

Comment:
validateExpression가 boolean 값을 반환하고 있으니 아래처럼 사용하면 어떨까요?

```suggestion
        if (!validateExpression(expression))
```

Reply:
이 부분은 일부로 이렇게 적은거긴한데, 별로일까요?
! 가 눈에 잘 안띄기도하고 식이 길어지더라도 `== false `가 글 읽는 순서대로 읽혀서 요렇게 사용하는 걸 선호하긴 합니다.

Reply:
이렇게 사용해도 무방하나, 비교 연산이 한번 더 발생한다는 점에서 좋은 방법인지는 고민되네요. 지금 미션은 혼자 진행하는거지만 사실 협업의 측면에서는 개인의 선호도보다는 협업하는 사람이나 일반적인 컨벤션을 고려하시는게 좋지 않을까 생각되어요.

그리고 !가 눈에 잘 안띈다면 isNot과 같이 메서드 이름을 추출하시는 것도 방법입니다.